<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cubic_gui</class>
 <widget class="QDialog" name="cubic_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EquateToGraph - Cubic Function</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../equatetograph.qrc">
    <normaloff>:/images/images/title_icon.png</normaloff>:/images/images/title_icon.png</iconset>
  </property>
  <widget class="QLabel" name="lbl_desc">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Enter the following values.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txt_cubic_out">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>521</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>521</width>
     <height>20</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="lbl_cubicformtext">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Form of Cubic Equation</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_cubiceq">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>y = ax^3 + bx^2 + cx + d</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>521</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="lbl_aValue">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>a</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_aValue"/>
    </item>
    <item>
     <widget class="QLabel" name="lbl_bValue">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>b</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_bValue"/>
    </item>
    <item>
     <widget class="QLabel" name="lbl_cValue">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>c</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_cValue"/>
    </item>
    <item>
     <widget class="QLabel" name="lbl_dValue">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>d</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_dValue"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_clear">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="btn_funcDetails">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Function Detais</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_discriminant">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Discriminant Value</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_CValue">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>C Value </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_drawGraph">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Draw Graph</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>9</y>
     <width>331</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCustomPlot" name="customPlot" native="true"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_saveImge">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Save As an Image</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../equatetograph.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_clear</sender>
   <signal>clicked()</signal>
   <receiver>line_aValue</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>546</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_clear</sender>
   <signal>clicked()</signal>
   <receiver>line_bValue</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>527</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_clear</sender>
   <signal>clicked()</signal>
   <receiver>line_cValue</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>524</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_clear</sender>
   <signal>clicked()</signal>
   <receiver>line_dValue</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>532</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_clear</sender>
   <signal>clicked()</signal>
   <receiver>txt_cubic_out</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>584</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>589</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
